modules/dnn/misc/python/test/test_dnn.py

```diff
@@ -286,38 +286,43 @@ def test_textdetection_model(self):
 
 
     def test_face_detection(self):
-        proto = self.find_dnn_file('dnn/opencv_face_detector.prototxt')
-        model = self.find_dnn_file('dnn/opencv_face_detector.caffemodel', required=False)
-        if proto is None or model is None:
-            raise unittest.SkipTest("Missing DNN test files (dnn/opencv_face_detector.{prototxt/caffemodel}). Verify OPENCV_DNN_TEST_DATA_PATH configuration parameter.")
+        model = self.find_dnn_file('dnn/onnx/models/yunet-202303.onnx')
```

Comment:
`modules/dnn/misc/python/test/test_dnn.py:105:    def find_dnn_file(self, filename, required=True)`
Please use `required` option for it. OpenCV CI uses `OPENCV_DNN_TEST_REQUIRE_TESTDATA=1` to not hide test failures. `OPENCV_TEST_DATA_PATH` is also set correctly, so the test should be executed on CI without issues.

Reply:
Thanks. I think it should check if the model exists, otherwise raise a `SkipTest`

Reply:
Thanks, that's a mistake.